Tests/WebSocketKitTests/WebSocketKitTests.swift

```diff
@@ -479,7 +451,9 @@ final class WebSocketKitTests: XCTestCase {
     }
     
     func testBadURLInWebsocketConnect() async throws {
-        XCTAssertThrowsError(try WebSocket.connect(to: "%w", on: self.elg, onUpgrade: { _ in }).wait()) {
+        // %w seems to now get to NIO and it attempts to connect to localhost:80 ... empty string makes the test pass
+        XCTAssertThrowsError(try WebSocket.connect(to: "", on: self.elg, onUpgrade: { _ in }).wait()) {
```

Comment:
Again, let's make sure we don't change tests just to get things passing.

Reply:
The test is no longer valid and thus did not pass when it in fact should have passed because it wasnt testing what it said it was testing. it expects %w to return nil not a URL if I recall. As a QA engineer I take tests pretty seriously.

Reply:
I'd add I tested this with a temporary test of URL init with both params and discerning what the test was testing, from that result and the test expectations. So I changed the setup to satisfy the expectation that existed prior.
I was surprised %w returned a URL and not nil.